test/standalone/font_registration_test.cpp

```diff
@@ -57,7 +57,7 @@ SECTION("registration") {
         mapnik::load_map_string(m3,"<Map font-directory=\"test/data/fonts/Noto/\"></Map>");
         REQUIRE( m3.get_font_memory_cache().size() == 0 );
         REQUIRE( m3.load_fonts() );
-        REQUIRE( m3.get_font_memory_cache().size() == 1 );
+        REQUIRE( m3.get_font_memory_cache().size() == 2 );
```

Comment:
For some reason latest 'data' module contains 2 fonts in the test/data/fonts/Noto folder, not 1.

Reply:
Oh, I did not get it before. But then in order to minimize changes probably the departing branch should be `3.0.x-carto`, which was cut from master at the time of releasing `3.0.15`.

Reply:
Perhaps it would be better to change the branch for test/data in .gitmodules to v3.0.15? Or are there additional tests in 3.0.x-carto branch missing from v3.0.15?

Reply:
I made this change:

https://github.com/CartoDB/mapnik/blob/v3.0.15-carto/CHANGELOG.carto.md#30153

https://github.com/CartoDB/mapnik/pull/26/files

https://github.com/CartoDB/mason/pull/1/files

https://github.com/CartoDB/test-data-visual/commit/2f22a10fbaa19dd12e078b39e3c2a9f579ea8c2c

because of a performance regression in harfbuzz